services/core-web/src/components/mine/Securities/MineReclamationInvoiceTable.js

```diff
@@ -207,7 +207,7 @@ export const MineReclamationInvoiceTable = (props) => {
     permits.map((permit) => {
       return {
         key: permit.permit_guid,
-        amount_confiscated: props.getSum("CON", permit),
+        amount_confiscated: props.getTotalConfiscatedSum(permit),
         amount_spent: props.getAmountSum(permit),
```

Comment:
if `permit.confiscated_bond_total` is on the permit, you can just reference that here 

`amount_confiscated: permit.confiscated_bond_total`

Reply:
Thats a fair point, I just wanted to abstract the implementation in case we go with something different in the future - but that's easy enough to change down the road.

Reply:
Updated